### ISSUE TITLE
Sessions not dying

### ISSUE THREAD

**Issue by careri** [Starcounter/Starcounter#3706]
When I hit F5 in the browser a new Starcounter.Session is created on the server.
I have a list of created sessions, that I sometimes call refresh upon. 

The strange thing is that all sessions seem to return IsAlive() even if they haven't been used for hours.
Any pointers to what I'm doing wrong?
From the VS debugger:
```
s={Starcounter.Session}, Starcounter.Session
s.IsAlive()=true, bool
s.IsBeingUsed()=true, bool
s.Created={2016-06-15 10:55:28}, System.DateTime
s.LastActive={2016-06-15 11:13:13}, System.DateTime
s.TimeoutMinutes=20, ulong
System.DateTime.Now={2016-06-15 13:16:27}, System.DateTime
```

**Comment by careri** [comment]
Eh, s.Created seems to use Zulu-time? It's two hours of because of the swedish summer time. That's strange?

**Comment by per-samuelsson** [comment]
> Eh, s.Created seems to use Zulu-time?

What, is that not where you live? :wink: 

Can you try [Session timeout](http://starcounter.io/guides/web/sessions/#sessions-timeout) and related callback, to see if it works as it is documented?

@chrhol @alemoi any ideas?

**Comment by per-samuelsson** [comment]
@careri 

Also don't forget all our host-level bits are open source: [Session](https://github.com/Starcounter/Starcounter/blob/6788776513bfd0f85ffe5f2600d149889fa967f4/src/Starcounter.XSON/Sessions/Session.cs).

**Comment by careri** [comment]
This gives me the time in my region.
```cs
var lastActive = s.LastActive.ToLocalTime();
```

Have a deadline tomorrow, can't investigate more right now

**Comment by alemoi** [comment]
Yes, the session creation time is now set as `s.Created = DateTime.UtcNow;`
Sessions timeout is based on internal counters though, so its not related to UTC time.
Should we save local time instead probably?

**Comment by per-samuelsson** [comment]
> Should we save local time in `Created` instead of UTC probably?

I'm thinking that the actual means we use to compare, retrieve and store times are not crucial as long as we provide a **consistent user experience**.

Looking at the output provided by @careri in the OP, I can totally understand the confusion, and we should aim to get rid of that.

It's this extract that seem confusing IMO:

```cs
s.LastActive={2016-06-15 11:13:13}, System.DateTime
s.TimeoutMinutes=20, ulong
s.IsAlive()=true, bool

System.DateTime.Now={2016-06-15 13:16:27}, System.DateTime
```

WTBS, someone else might compare using `DateTime.UtcNow`, and if we adapt to local time, such comparision will look equally confusing. So it's not a clear right or wrong here.

What about that we instead add a property that include the "current time" considered by the session? By doing that, `IsAlive()` will be easier to understand.

```cs
s.LastActive={2016-06-15 11:13:13}, System.DateTime
s.CurrentTime={2016-06-15 11:17:55}, System.DateTime
s.TimeoutMinutes=20, ulong
s.IsAlive()=true, bool
```

Another alternative: we don't expose any time values at all, but instead just either a property that shows for how long the session has lived, or one that shows how long it is until it's dead. That way, the timing we use is in implementation detail.

```cs
s.Lifetime={00:44:25}, System.TimeSpan // (total lifetime, 44 minutes 25 seconds since created)
s.LastActive={00:2:13}, System.TimeSpan // 2 minutes 13 seconds since last active
s.TimeoutMinutes=20, ulong
s.IsAlive()=true, bool  // Now this make sense
```

WDYT? @alemoi

**Comment by alemoi** [comment]
@per-samuelsson thank you for attention on this! I like the last suggestion about living active time.
